### PR TITLE
Turning on Kitchen PR CI

### PR DESCRIPTION
**Pull request by gitromba**
Integrating repo to Kitchen PR CI pipeline by adding .Jenkinsfile from tao-core.

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
     'label' => 'QTI Portable Custom Interaction',
     'description' => '',
     'license' => 'GPL-2.0',
-    'version' => '6.6.1',
+    'version' => '6.7.1',
```
Comment (bugalot) [inline manifest.php:37]: Should be `6.7.0`

File: .Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                                label : 'Run frontend tests',
+                                script: 'npx grunt connect:test qtiitempcitest'
+                            )
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (bugalot) [inline .Jenkinsfile:128]: Please add end of line.

File: scripts/update/Updater.php [hunk 3]
```diff
@@ -428,5 +428,7 @@ public function update($currentVersion)
         }
 
         $this->skip('6.6.0', '6.6.1');
+
+        $this->skip('6.6.1', '6.7.1');
```
Comment (bugalot) [inline scripts/update/Updater.php:432]: A new `$this->skip()` call is not necessary. You can change the existing `skip('6.6.0', '6.6.1')` to `skip('6.6.0', '6.7.0')`.

Comment (gitromba) [reply scripts/update/Updater.php:432]: Nice trick, thanks @bugalot !

File: .Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                        HOME = '.'
+                    }
+                    options {
+                        skipDefaultCheckout()
+                    }
+                    steps {
+                        dir('build/tao/views/build') {
+                            sh(
+                                label: 'Install tao-core frontend extensions',
+                                script: 'npm install'
```
Comment (bugalot) [inline .Jenkinsfile:116]: Something interesting: 

In the `tao-core` extension's `.JenkinsFile`, the same line is actually `script: 'npm install --production'`. We need to understand what is the difference and whether or not we need `--production`.

Comment (gitromba) [reply .Jenkinsfile:116]: It's like `no-dev` for composer. `--production` -> no devdepencies are installed.

**Pull request merged**